satpy/readers/viirs_sdr.py

```diff
@@ -275,9 +275,18 @@ def get_dataset(self, dataset_id, ds_info):
         data.attrs.update(i)
         return data
 
+    def get_bounding_box(self):
```

Comment:
I could see this not working just because of all the weird ways VIIRS SDRs end up being formatted/organized. We'll have to find out later.

Reply:
Is this in the base file handler? Does it have information on what needs to get returned?

Reply:
It is in the base filehandler, raising a NotImplementedError. It's not documented as for now, though the name is quite self-explanatory I believe.

Reply:
The clockwise part isn't self-explanatory. And I could also see someone thinking that a bounding box is min lon, max lon, min lat, max lat.

Reply:
Added now some documentation to the abstract base class.